app/src/seed_script.py

```diff
@@ -0,0 +1,195 @@
+import os
+from .models import db, Course, Module, Topic, Lesson, Subject, User, Points, UserProgress, GradeEnum
+
+def create_file_paths(course_id, module_id, topic_id):
+    # This creates the file path for the video/thumbnails of a given course, still need to manually add in the video/
+    # thumbnails into the created directory
+
+    script_dir = os.path.dirname(os.path.abspath(__file__))
+
+    lesson_video_path = os.path.join(script_dir, f"static/vendor/lesson-videos/{course_id}-{module_id}-{topic_id}")
+    textbook_path = os.path.join(script_dir, f"static/vendor/textbooks/{course_id}")
+    
+    print("Lesson Video Path:", lesson_video_path)
+    print("Textbook Path:", textbook_path)
+
+    os.makedirs(lesson_video_path, exist_ok=True)
+    os.makedirs(textbook_path, exist_ok=True)
```

Comment:
Replace these lines with the following to copy over placeholder files to the new directories you just created, so the lessons will actually render the appropriate content:

```python
os.makedirs(textbook_path, exist_ok=True)
copyfile(os.path.join(script_dir, TEXTBOOK_PLACEHOLDER), os.path.join(textbook_path, "sample.pdf"))

if is_video_lesson:
    os.makedirs(lesson_video_path, exist_ok=True)
    copyfile(os.path.join(script_dir, VIDEO_PLACEHOLDER), os.path.join(lesson_video_path, "video.mp4"))
    copyfile(os.path.join(script_dir, THUMBNAIL_PLACEHOLDER), os.path.join(lesson_video_path, "thumbnail.png"))
```